README.md

```diff
@@ -1,6 +1,7 @@
 # Google Student Training in Engineering Program
+Updated: 5/19
 
-This repo contains the projects you'll work on for the first half of STEP.
+This repo contains the projects I will on for the first half of STEP.
```

Comment:
nit: I think this should be "... I will work ..."

Reply:
Yes, I ignored this typo, thank you!